custom/icds_reports/tests/agg_tests/outputs/agg_awc_2017-05-01_5_sorted.csv

```diff
@@ -1,56 +1,56 @@
tival,awc_not_open_no_help,awc_not_open_department_work,awc_not_open_other,awc_num_open,awc_not_open_no_data,wer_weighed,wer_eligible,wer_score,thr_eligible_child,thr_rations_21_plus_distributed_child,thr_eligible_ccs,thr_rations_21_plus_distributed_ccs,thr_score,awc_score,num_awc_rank_functional,num_awc_rank_semi,num_awc_rank_non,cases_ccs_pregnant,cases_ccs_lactating,cases_child_health,usage_num_pse,usage_num_gmp,usage_num_thr,usage_num_home_visit,usage_num_bp_tri1,usage_num_bp_tri2,usage_num_bp_tri3,usage_num_pnc,usage_num_ebf,usage_num_cf,usage_num_delivery,usage_num_due_list_ccs,usage_num_due_list_child_health,usage_awc_num_active,usage_time_pse,usage_time_gmp,usage_time_bp,usage_time_pnc,usage_time_ebf,usage_time_cf,usage_time_of_day_pse,usage_time_of_day_home_visit,vhnd_immunization,vhnd_anc,vhnd_gmp,vhnd_num_pregnancy,vhnd_num_lactating,vhnd_num_mothers_6_12,vhnd_num_mothers_12,vhnd_num_fathers,ls_supervision_visit,ls_num_supervised,ls_awc_location_long,ls_awc_location_lat,ls_awc_present,ls_awc_open,ls_awc_not_open_aww_not_available,ls_awc_not_open_closed_early,ls_awc_not_open_holiday,ls_awc_not_open_unknown,ls_awc_not_open_other,infra_last_update_date,infra_type_of_building,infra_type_of_building_pucca,infra_type_of_building_semi_pucca,infra_type_of_building_kuccha,infra_type_of_building_partial_covered_space,infra_clean_water,infra_functional_toilet,infra_baby_weighing_scale,infra_flat_weighing_scale,infra_adult_weighing_scale,infra_cooking_utensils,infra_medicine_kits,infra_adequate_space_pse,usage_num_hh_reg,usage_num_add_person,usage_num_add_pregnancy,is_launched,training_phase,trained_phase_1,trained_phase_2,trained_phase_3,trained_phase_4,aggregation_level,num_launched_states,num_launched_districts,num_launched_blocks,num_launched_supervisors,num_launched_awcs,cases_household,cases_person,cases_person_all,cases_person_has_aadhaar,cases_ccs_pregnant_all,cases_ccs_lactating_all,cases_child_health_all,cases_person_adolescent_girls_11_14,cases_person_adolescent_girls_15_18,cases_person_adolescent_girls_11_14_all,cases_person_adolescent_girls_15_18_all,infra_infant_weighing_scale,cases_person_beneficiary,cases_person_referred,awc_days_pse_conducted,num_awc_infra_last_update,cases_person_has_aadhaar_v2,cases_person_beneficiary_v2,electricity_awc,infantometer,stadiometer,num_anc_visits,num_children_immunized,state_is_test,district_is_test,block_is_test,supervisor_is_test,awc_is_test,num_awcs_conducted_cbe,num_awcs_conducted_vhnd,valid_visits,expected_visits,wer_eligible_0_2,wer_weighed_0_2,thr_distribution_image_count,num_mother_thr_eligible,num_mother_thr_21_days,toilet_facility,type_toilet,preschool_kit_available,preschool_kit_usable,awc_with_gm_devices,cases_person_adolescent_girls_11_14_all_v2,cases_person_adolescent_girls_11_14_out_of_school,cases_ccs_lactating_reg_in_month,cases_ccs_pregnant_reg_in_month,cases_ccs_lactating_all_reg_in_month,cases_ccs_pregnant_all_reg_in_month,cbe_conducted,vhnd_conducted,app_version,commcare_version
```

Comment:
why are these `agg_awc` test case gets updated? I could not find any change in `agg_awc.py`

Reply:
removed it. They are from the earlier approach